deploy.php

```diff
@@ -0,0 +1,413 @@
+<?php
+
+namespace Deployer;
+
+use Symfony\Component\Yaml\Yaml;
+
+require 'recipe/symfony3.php';
+require 'recipe/cachetool.php';
+require 'recipe/sentry.php';
+require __DIR__ . '/vendor/tijsverkoyen/deployer-sumo/sumo.php';
+
+// Define some variables
+set('client', '$client');
+set('project', '$project');
+set('repository', '$repository');
+set('user', '$user');
+set('release', 'current');
+set('production_url', '$productionUrl');
+set('sentry_organization', '$sentryOrganization');
+set('sentry_project_slug', '$sentryProjectSlug');
+set('sentry_token', '$sentryToken');
+
+// Define staging
+host('dev03.sumocoders.eu')
+    ->user('{{user}}')
+    ->stage('staging')
+    ->set('deploy_path', '~/apps/{{client}}/{{project}}')
+    ->set('branch', 'staging')
+    ->set('bin/php', 'php7.4')
+    ->set('bin/composer', '{{bin/php}} /home/sites/apps/{{client}}/{{project}}/shared/composer.phar')
```

Comment:
Volgens mij is dat niet nodig, zie https://github.com/deployphp/deployer/blob/6.x/recipe/common.php#L124

Reply:
Zie https://github.com/sumocoders/forkcms/pull/249/commits/5c30c7a8a3b65a360f22c15a841748452240c944 en https://github.com/sumocoders/forkcms/pull/249/commits/5f8cd24d6df2298413c633237f80d8b6322e28d0